cmd/utils/flags.go

```diff
@@ -954,6 +954,12 @@ Please note that --` + MetricsHTTPFlag.Name + ` must be set to start the server.
 		Value:    metrics.DefaultConfig.InfluxDBOrganization,
 		Category: flags.MetricsCategory,
 	}
+
+	CollectWitnessesFlag = &cli.BoolFlag{
+		Name:  "collectwitnesses",
```

Comment:
Please name this `debug.collectwitness` and move it to the debug namespace in the usage section so we know it's not something to rely on.

As for the description, perhaps: `Enable state witness generation during block execution. Work in progress flag, don't use.`

Reply:
Also Pls rename the flag to DebugCollectWitness